spec/support/marmotta_clear.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.before(:each) do
```

Comment:
I timed it before and after and the timing didn't change. Seems like the same kind of thing as DatabaseCleaner.